protocols/cosi/cosi.go

```diff
@@ -323,14 +323,14 @@ func (pc *ProtocolCosi) Cleanup() {
 		pc.DoneCallback(pc.Cosi.GetChallenge(), pc.Cosi.GetAggregateResponse())
 	}
 	close(pc.done)
-	pc.Node.Done()
+	pc.TreeNodeInstance.Done()
```

Comment:
I just changed from `Node` to `TreeNodeInstance`. But I did not check if we registered a callback on TreeNodeInstance; and we don't so I changed to it `pc.Done()`